### PR TITLE
Feature/spacing as a property

### PR DESCRIPTION
**Pull request by axenovV**

### CODE REVIEW

File: CBPinEntryView/Classes/CBPinEntryViewDefaults.swift [hunk 1]
```diff
@@ -12,6 +12,9 @@ struct CBPinEntryViewDefaults {
 
     // Default number of fields
     static let length: Int = 4
+    
+    //Default spacing between fileds
```
Comment (Fawxy) [inline CBPinEntryView/Classes/CBPinEntryViewDefaults.swift]: Can you change this comment to `// Default spacing between fields`

Comment (axenovV) [reply CBPinEntryView/Classes/CBPinEntryViewDefaults.swift]: Yes, of course. I am sorry

Comment (Fawxy) [reply CBPinEntryView/Classes/CBPinEntryViewDefaults.swift]: Also the spelling of fields :)

Comment (axenovV) [reply CBPinEntryView/Classes/CBPinEntryViewDefaults.swift]: @Fawxy I fixed it

**Pull request merged**